ansible_base/oauth2_provider/authentication.py

```diff
@@ -42,8 +42,14 @@ def authenticate(self, request):
         if ret:
             user, token = ret
             username = user.username if user else '<none>'
+            oauth2_application_pk = token.application.pk if token.application else "N/A"
+            oauth2_application_name = token.application.name if token.application else "Personal Access Token"
             logger.info(
-                smart_str(u"User {} performed a {} to {} through the API using OAuth 2 token {}.".format(username, request.method, request.path, token.pk))
+                smart_str(
+                    u"User {} performed a {} to {} through the API using OAuth 2 token {} for OAuth2 application {} ({}).".format(
```

Comment:
Consider using `log_auth_event` here for consistency with the access_token.py and refresh_token.py changes. This would ensure all OAuth2-related events are logged to the audit logger.

Note: This would require importing `log_auth_event` from `ansible_base.lib.logging`.